<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTumbler</class>
 <widget class="QWidget" name="frmTumbler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Tumbler" name="tumbler1" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>211</width>
     <height>90</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="horizontal" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Tumbler" name="tumbler2" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>105</y>
     <width>211</width>
     <height>90</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="horizontal" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Tumbler" name="tumbler3" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>201</y>
     <width>211</width>
     <height>90</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="horizontal" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Tumbler" name="tumbler4" native="true">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>9</y>
     <width>84</width>
     <height>282</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="Tumbler" name="tumbler5" native="true">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>9</y>
     <width>85</width>
     <height>282</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="Tumbler" name="tumbler6" native="true">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>9</y>
     <width>84</width>
     <height>282</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tumbler</class>
   <extends>QWidget</extends>
   <header>tumbler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
